excellent/legacy.go

```diff
@@ -175,6 +175,12 @@ func newVars() vars {
 				vars:    contactVars,
 			},
 		},
+		"channel": vars{
+			"__default__": "contact.channel.address",
+			"name":        "contact.channel.name",
+			"tel":         "contact.channel.address",
+			"tel_e164":    "contact.channel.address",
```

Comment:
We have the same thing on contact. I think it is just a matter of e164 addresses not being terribly friendly  if you are including it in a message, thus the tel version. We could maybe add a new excellent method to do the translation instead? But that isn't super friendly either.